odoo/openupgrade/doc/source/modules110-120.rst

```diff
@@ -638,6 +638,10 @@ missing in the new release are marked with |del|.
 |website_twitter                             |                                                 |
 +--------------------------------------------+-------------------------------------------------+
 
+.. [#account_asset] 'Account Asset' Module is now under Odoo Community Association Umbrella
```

Comment:
I would rephrase this to say that it's alternative module. Right now, it seems like other forward-portings done like `crm_claim`.